tests/TestUtil.elm

```diff
@@ -0,0 +1,34 @@
+module TestUtil exposing (range, stringToIntermediate)
+
+{-| Contains utility functions for the test suite.
+-}
+
+import Elm.Parser
+import Elm.Processing
+import Elm.Syntax.Range
+import Intermediate
+import Models
+
+
+{-| Parses a string to a ParsedModule.
```

Comment:
please add a description of "a string" -- that is assumed to be valid Elm code, right?